build.sbt

```diff
@@ -169,7 +169,7 @@ lazy val googleCloudPubSubGrpc = alpakkaProject(
   javaAgents += Dependencies.GooglePubSubGrpcAlpnAgent % Test,
   // for the ExampleApp in the tests
   connectInput in run := true,
-  scalacOptions += "-P:silencer:pathFilters=src_managed",
+  scalacOptions in (Compile, compile) += "-P:silencer:pathFilters=src_managed",
```

Comment:
I suggest to use the `Compile / compile / scalacOptions` notation, as we do in most places now.